org/Mini_SAM/index.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: org
+title: Benjamin Shockley
```

Comment:
This file needs to briefly describe your organisation - or yourself if it's just you, and should be named accordingly. The PID file is where the description of the specific device should go.